pymatbridge/version.py

```diff
@@ -1,90 +0,0 @@
-"""pymatbridge version/release information"""
-
-# Format expected by setup.py and doc/source/conf.py: string of form "X.Y.Z"
-_version_major = 0
-_version_minor = 5
-_version_micro = 0 #''  # use '' for first of series, number for 1 and above
-_version_extra = 'dev'
-#_version_extra = ''  # Uncomment this for full releases
-
-# Construct full version string from these.
-_ver = [_version_major, _version_minor]
-if _version_micro:
-    _ver.append(_version_micro)
-if _version_extra:
-    _ver.append(_version_extra)
-
-__version__ = '.'.join(map(str, _ver))
-
-CLASSIFIERS = ["Development Status :: 3 - Alpha",
-               "Environment :: Console",
-               "Intended Audience :: Science/Research",
-               "License :: OSI Approved :: BSD License",
-               "Operating System :: OS Independent",
-               "Programming Language :: Python",
-               "Topic :: Scientific/Engineering"]
-
-description = "pymatbridge is a set of python and matlab functions to allow these two systems to talk to each other"
-
-long_description = """
```

Comment:
Will the long_description still be pulled into the pypi page using the information from the `setup.cfg` file? That seems like a plus - no need to replicated information in the README and here